server/services/random.js

```diff
@@ -1,3 +1,6 @@
+const minResourceScore = 10;
+const maxResourceScore = 50;
+
```

Comment:
also we can add const resourceRange in here nad i would rename it to RESOURCE_DLETA (delta is almost everytime in math max - min)

Reply:
Also add here constants RS_BASE and RS_EXPONENT 

```
const RS_BASE = 2
const RS_EXPONENT = 5.8
```